ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js

```diff
@@ -74,11 +74,12 @@ export default class DepositEtherModal extends Component {
     )
   }
 
-  render () {
+  render() {
     const { network, toFaucet } = this.props
-
-    const isTestNetwork = ['3', '4', '5', '42'].find((n) => n === network)
```

Comment:
The network IDs and chain IDs are the same just hex equivalent of decimal values:
```
export const MAINNET_NETWORK_ID = '1'
export const ROPSTEN_NETWORK_ID = '3'
export const RINKEBY_NETWORK_ID = '4'
export const GOERLI_NETWORK_ID = '5'
export const KOVAN_NETWORK_ID = '42'

export const MAINNET_CHAIN_ID = '0x1'
export const ROPSTEN_CHAIN_ID = '0x3'
export const RINKEBY_CHAIN_ID = '0x4'
export const GOERLI_CHAIN_ID = '0x5'
export const KOVAN_CHAIN_ID = '0x2a'
```

So I think the problem is that we made the network an integer now and it doesn't match the strings.
I think you can change the test to be `[3, 4, 5, 42].find((n) => n === network)` and it might work?

If I'm right then below you'd pass `toFaucet(network.toString)`